src/yunohost/domain.py

```diff
@@ -346,6 +346,10 @@ def _build_dns_conf(domain, ttl=3600):
             ["*", ttl, "AAAA", ipv6],
         ]
 
+    basic += [ 
```

Comment:
Rethinking about this today : I definitely think we should add a new category called "extra" since this ... extra stuff which is not required for things to work. Also this should simplify the integration with `dyndns update` (the dynette might probably refuse to add this record right now, or we need to edit it) and simplify other things I anticipate for the auto-diagnosis